src/components/mainlayout.tsx

```diff
@@ -0,0 +1,16 @@
+import { Sidebar } from "./sidebar";
+
+type MainLayoutProps = {
+  children?: React.ReactNode;
+};
+
+export const MainLayout = (props: MainLayoutProps): React.JSX.Element => {
```

Comment:
I suggest you change from "export const MainLayout = () => {}" to "export function MainLayout() {}", because I noticed that in the rest of the application you are using this pattern and only in this component you are using this way of creating a function. OR, if you prefer to export functions like this, then change everywhere else to this pattern. Remember: be consistent!